### ISSUE TITLE
Process explorer's highest resource does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51823]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
ColorContrast,MAS1.4.3

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Help(Alt+H) menu option and select "Open Process Explorer" from dropdown.
3. Open color contrast analyzer and check the ratio.

**Actual:**
Color contrast ratio is less than 4.5:1 for the digits which are dynamically changes.

**Expected:**
Color Contrast ratio should not be less than 4.5:1.
Same issue observed on other color theme also.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
The user will not be able to see the controls.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
**Attachment for Reference:** 

![accessibility a11y_vscode_processexplorer_colorcontrast](https://user-images.githubusercontent.com/39204909/41392921-f7d5310a-6fc0-11e8-926b-ac3b79820faf.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I feel like this is low hanging fruit -> assigning to July and also adding @misolori

**Comment by RMacfarlane** [comment]
I tested the color contrast ratio of the two colors (`editor.background` and `list.highlightForeground`) with the default light and dark themes and they meet the requirement:

![screen shot 2018-07-05 at 11 53 11 am](https://user-images.githubusercontent.com/3672607/42342099-058c6af4-804a-11e8-8c05-67e446200932.png)

![screen shot 2018-07-05 at 11 55 22 am](https://user-images.githubusercontent.com/3672607/42342243-6bcbbb44-804a-11e8-9824-87c1b4b89776.png)

Since we're scoping to only those themes, I'll go ahead and close this
